src/Stratis.SmartContracts.CLR.Validation/Stratis.SmartContracts.CLR.Validation.csproj

```diff
@@ -9,12 +9,9 @@
 
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="0.11.4" />
+    <PackageReference Include="Stratis.SCL" Version="2.0.3-PreRelease" />
```

Comment:
Wouldn't this project already depend on the 2.0.3-PreRelease version of `Stratis.SCL` by virtue of the project reference? It seems messy to specify a package reference like this within the same solution as the package's project